packages/react/slot/src/Slot.tsx

```diff
@@ -5,7 +5,7 @@ import { composeRefs } from '@radix-ui/react-compose-refs';
  * Slot
  * -----------------------------------------------------------------------------------------------*/
 
-type SlotProps = { children: React.ReactNode };
+type SlotProps = { children?: React.ReactNode };
```

Comment:
What was going wrong under the hood here? I looked at the linked sandbox but the polymorphic errors are a bit cryptic and I'm unclear on how this change resolves it 😅